### PR TITLE
add center net docs

### PR DESCRIPTION
**Pull request by zhreshold**
Adding docs and results to website

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1057-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1057/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1057-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1057/2/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1057-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1057/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by zhreshold** [comment]
@hetong007 @bryanyzhu @Jerryzcn

File: docs/model_zoo/detection.rst [hunk 1]
```diff
@@ -126,6 +126,33 @@ Checkout YOLO demo tutorial here: :ref:`sphx_glr_build_examples_detection_demo_y
    | yolo3_mobilenet1.0_voc [3]_ :gray:`(416x416)`| 75.8  | `shell script <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/detection/yolo3_mobilenet1.0_voc.sh>`_            | `log <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/detection/yolo3_mobilenet1.0_voc.log>`_                   |
    +----------------------------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
 
+CenterNet
+---------
+
+CenterNet models are evaluated at 512x512 resolution. mAPs with flipped inference(F) are also reported, however, the models are identical.
+Checkout CenterNet demo tutorial here: :ref:`sphx_glr_build_examples_detection_demo_center_net.py`
+
+Note that ``dcnv2`` indicate that models include Modulated Deformable Convolution (DCNv2) layers, you may need to upgrade MXNet in order to use them.
```
Comment (bryanyzhu) [inline docs/model_zoo/detection.rst]: Do we need to specify which version of MXNet do they need to upgrade to?

File: gluoncv/utils/__init__.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 from .download import download, check_sha1
 from .filesystem import makedirs
 from .filesystem import try_import_dali
+from .filesystem import try_import_cv2
```
Comment (hetong007) [inline gluoncv/utils/__init__.py]: can we have all `filesystem` functions imported in the same lin (which i believe is perferred by pylint)?

File: docs/model_zoo/detection.rst (hunk 1 above)
Comment (zhreshold) [reply docs/model_zoo/detection.rst]: It's still a nightly version so I won't include it now, I can wait for a release version of mxnet

File: gluoncv/utils/__init__.py (hunk 2 above)
Comment (zhreshold) [reply gluoncv/utils/__init__.py]: Fixed

**Comment by mli** [comment]
Job PR-1057-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1057/4/index.html 
Code coverage of this PR:  vs. Master:

**Pull request merged**